lib/PromptSession.ts

```diff
@@ -21,71 +21,68 @@ export class PromptSession {
 
 		add.templateManager = this.templateManager;
 
```

Comment:
// tslint:disable:object-literal-sort-keys should be added back in order tslint to pass